cartographer/mapping/map_builder_interface.h

```diff
@@ -61,8 +61,8 @@ class MapBuilderInterface {
   // Returns the TrajectoryBuilder corresponding to the specified
   // 'trajectory_id' or 'nullptr' if the trajectory has no corresponding
   // builder.
-  virtual mapping::TrajectoryBuilder* GetTrajectoryBuilder(
-      int trajectory_id) const = 0;
+  virtual mapping::GlobalTrajectoryBuilderInterface *
+  GetTrajectoryBuilder(int trajectory_id) const = 0;
```

Comment:
The method name can stay the same if we are planning to drop "Global" from GlobalTrajectoryBuilderInterface. Otherwise, rename.

Reply:
Sounds good.